examples/gno.land/p/demo/ujson/parse.gno

```diff
@@ -0,0 +1,569 @@
+package ujson
+
+import (
+	"std"
+	"strconv"
+	"strings"
+
+	"gno.land/p/demo/avl"
+)
+
+// https://stackoverflow.com/a/4150626
+const whitespaces = " \t\n\r"
+
+type FromJSONAble interface {
+	FromJSON(ast *JSONASTNode)
+}
+
+// does not work for slices, use ast exploration instead
+func (ast *JSONASTNode) ParseAny(ptr *interface{}) {
```

Comment:
`*interface{}` is not what you think it is, you probably want to use `interface{}` here

Reply:
can you elaborate? I was thinking this was enforcing that I get a pointer as argument so I can assign to what it's pointing at